lib/stylelint-vscode/index.js

```diff
@@ -312,6 +311,12 @@ async function resolveStylelint({
 		// ignore
 	}
 
+	if (!stylelint) {
+		connection?.console.error(
+			'Failed to load stylelint either globally or from the current workspace.',
```

Comment:
What this looks like:

![screenshot](https://user-images.githubusercontent.com/9868643/135668942-8d898d77-f5ff-40df-91c6-b285deec6125.png)